e2e_playwright/shared/pydeck_utils.py

```diff
@@ -66,7 +66,7 @@ def wait_for_chart(app: Page):
     # The pydeck chart takes a while to load so check that
     # it gets attached with an increased timeout.
     pydeck_charts = app.get_by_test_id("stDeckGlJsonChart")
-    expect(pydeck_charts).to_have_count(5, timeout=15000)
+    expect(pydeck_charts).to_have_count(1, timeout=15000)
```

Comment:
nit: since we are only waiting for one chart.. maybe the timeout can be lowered a little bit